pkg/database/vercode.go

```diff
@@ -69,12 +68,15 @@ type VerificationCode struct {
 	gorm.Model
 	Errorable
 
-	RealmID       uint   // VerificationCodes belong to exactly one realm when issued.
-	Code          string `gorm:"type:varchar(512)"`
-	LongCode      string `gorm:"type:varchar(512)"`
-	UUID          string `gorm:"type:uuid;default:null"`
-	Claimed       bool   `gorm:"default:false"`
-	TestType      string `gorm:"type:varchar(20)"`
+	// RealmID is the realm in which this verification code was issued. Codes belong to exactly one realm.
+	RealmID uint `gorm:"column:realm_id; type:integer; not null;"`
+	Realm   *Realm
```

Comment:
I'm not sure the loading of the realm relation is 100% necessary here...

Reply:
Done